beaver/transports/redis_transport.py

```diff
@@ -75,13 +76,23 @@ def callback(self, filename, lines, **kwargs):
             namespace = self._namespace
         self._logger.debug('Got namespace: ' + namespace)
 
+        data_type = self._data_type
+        self._logger.debug('Got data type: ' + data_type)
+
         server = self._get_next_server()
         self._logger.debug('Got redis server: ' + server['url'])
 
         pipeline = server['redis'].pipeline(transaction=False)
 
+        if data_type == 'list':
+            data_type_method = pipeline.rpush
+        elif data_type == 'channel':
+            data_type_method = pipeline.publish
+        else:
+            raise TransportException('Unknown Redis data type')
```

Comment:
Tried to address in https://github.com/hectcastro/python-beaver/commit/39ee609fd9e4d273eb9f3b68a09514c6f9dedf45. Let me know what you think.